README.md

```diff
@@ -266,6 +266,36 @@ with AthenaDriver Config: 123
 
 The full code is here at [examples/auth.go](https://github.com/uber/athenadriver/tree/master/examples/auth.go).
 
+#### Use AWS SDK Default Credentials Resolution for Authentication
+If environment variable `AWS_SDK_LOAD_CONFIG` is NOT set and credentials are not supplied in the athenadriver configuration, the AWS SDK will lookup credentials using its default methodology described here: https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials.
```

Comment:
Nit:

```suggestion
If environment variable `AWS_SDK_LOAD_CONFIG` is NOT set and credentials are not supplied in the `athenadriver` configuration, the AWS SDK will look up credentials using its default methodology described here: https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials.
```

Reply:
This is an eventual PR back to the source library (pending OSS approval), so nitpicking is great.